src/pkg/library/client/search.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package client
+
+import (
+	"fmt"
+)
+
+// SearchLibrary will search the library for a given query ans display results
+func SearchLibrary(value string, libraryURL string, authToken string) error {
+	if len(value) < 3 {
+		return fmt.Errorf("Bad query '%s'. You must search for at least 3 characters", value)
```

Comment:
with only 2 characters you don't actually get a huge number of discriminating pairs - since many 2 character combinations don't (commonly or at all) occur in words... e.g. for English see:

 http://homepages.math.uic.edu/~leon/mcs425-s08/handouts/char_freq2.pdf

If you have a strong preference for 2 we could do that here in the CLI where a search operation is a one time thing. I wouldn't want to use 2 for autocomplete on the web where the request gets fired over and over.

Reply:
Oh totally.  This should not be part of this PR.  Just wanted to chat a bit and get some insight.